test/import-map.test.js

```diff
@@ -0,0 +1,106 @@
+const request = require("supertest");
+const { app, setConfig } = require("../src/web-server");
+
+beforeAll(() => {
+  setConfig({
+    manifestFormat: "importmap",
+    username: "username",
+    password: "password",
+    locations: {
+      prod: "memory://prod",
+    },
+  });
+});
+
+describe(`/import-map.json`, () => {
+  it(`does not return anything when it's not setup yet.`, async () => {
+    const healthResponse = await request(app)
+      .get("/import-map.json")
+      .expect(200)
+      .expect("Content-Type", /json/);
+
+    // we did not setup yet
+    expect(healthResponse.body.message).toBe(undefined);
+  });
+
+  it(`does give back the same items after first patch`, async () => {
+    const response = await request(app)
+      .patch("/import-map.json")
+      .query({
+        skip_url_check: true,
+      })
+      .set("accept", "json")
+      .send({
+        imports: {
+          a: "/a-1.mjs",
+          b: "/b-1.mjs",
+          c: "/c-1.mjs",
+        },
+      })
+      .expect(200)
+      .expect("Content-Type", /json/);
+
+    // we did not setup yet
+    expect(response.body).toMatchObject({
+      imports: {
+        a: "/a-1.mjs",
+        b: "/b-1.mjs",
+        c: "/c-1.mjs",
+      },
+      scopes: {},
+    });
+  });
+
+  it(`The data is still in there`, async () => {
```

Comment:
This test relies on test execution order, which makes tests very brittle. We should not require the test on line 26 to execute before this test in order for this test to pass. Each test should be independent.